source/reference/minio-mc/mc-mirror.rst

```diff
@@ -191,44 +173,61 @@ Parameters
    
    Overwrites object(s) on the :mc-cmd:`~mc mirror TARGET`.
 
+   For example, object A exists on both the Source and the Target.
+   Object A on the source changes.
+   Running `mc mirror --overwrite` replaces the version on the target with the version from the Source.
+
```

Comment:
We should restructure this in context of `--overwrite` already running, and then a change occurring:

```suggestion
   For example, consider an active `mc mirror --overwrite` synchronizing content from source to destination.
   
   If an object on the source changes, `mc mirror` synchronizes and overwrites any matching file on the destination.
   
   Without --overwrite, <behavior>

```

The inclusion of *version* here also might need some care. Buckets can also be unversioned, in which case I assume it's just a straight overwrite.